src/main/java/grandduke/command/Parser.java

```diff
@@ -0,0 +1,108 @@
+package grandduke.command;
+
+import grandduke.task.TaskList;
+import grandduke.task.Deadline;
+import grandduke.task.Event;
+import grandduke.task.Task;
+import grandduke.task.Todo;
+
+public abstract class Parser {
+    /**
+     * Parses the command from the user and decides on the appropriate command to
+     * execute
+     * 
+     * @param input
+     *            the command sent by the user
+     */
+    public static void parseCommand(String input) {
+        input = input.toLowerCase();
+        String[] inputArr = input.split(" ", 2);
+
+        if (input.equals(Io.LIST_COMMAND)) {
+            TaskList.printTaskList();
+        } else if (input.startsWith(Io.MARK_COMMAND)) {
```

Comment:
Perhaps this can be replaced with a switch case statement, for every statement besides the first and last? I think it will improve the readability

Reply:
Thanks for the suggestion, I was just thinking about doing this change because I also felt it was ugly